src/app/records/Record.m.scss

```diff
@@ -128,10 +128,7 @@
 
 .trackedInDim .glow {
   display: block;
-  background: linear-gradient(
-    scale-color($orange, $alpha: -100%) 0%,
-    scale-color($orange, $alpha: -80%) 100%
-  );
+  background: linear-gradient(transparent 0%, var(--theme-accent-primary) 500%);
```

Comment:
This does not seem the same?

Reply:
Granted, I did eye-ball this — but it looks the same to me? 

Left: this commit. Right: existing gradient. 

![image](https://user-images.githubusercontent.com/1396158/236188524-42e0a744-a4cb-4418-9704-a036a8512fa9.png)

Reply:
OK, I get it. Another case of not being able to adjust alpha on a color defined in a var.